docs/Framework.md

```diff
@@ -311,7 +311,7 @@ NcClassId is a sequence of NcInt32 class ID fields.
 A class ID sequence reflects the ancestry of the class being identified.
 
 A class ID field is either a definition index or an authority key.
-A definition index is an ordinal that starts at 1 for every inheritance level of the control model class tree for example `[ 1, 1, 3, 5]`.
+A definition index is an ordinal that starts at 1 for every inheritance level of the control model for example `[ 1, 1, 3, 5]`.
```

Comment:
@cristian-recoseanu Am I being overzealous here in removing the "class tree" here?  I think that "control model class tree" is probably more descriptive in this context.  My suggestion is to put "class tree" back.
```suggestion
A definition index is an ordinal that starts at 1 for every inheritance level of the control model class tree for example `[ 1, 1, 3, 5]`.
```